mockdata/facebookMockData.js

```diff
@@ -0,0 +1,16 @@
+export const profile = {
+  id: 92647913649816349,
+  emails: [{
+    value: 'chris@yahoo.com'
+  }],
+  name: {
+    givenName: 'Christopher',
+    familyName: 'Akanmu'
+  },
+  photos: [{
+    value: 'https://facebook.com/profilepicture'
+  }]
+};
+
+export const accessToken = 'khgsda6d8tyi3bu97bfepiohnfl2n3pf0hoql.23 fnkiu3g';
+export const refreshToken = 'khsgad63iy789r32vyf923fv23o0fig2o3f';
```

Comment:
Nice work Chris, but shouldn't we have these tokens saved as environment variables?

Reply:
@mrSerious Thanks. No, these are dummy strings usually returned to the callback function by facebook. They are not really private data.